screengrab/lib/screengrab/runner.rb

```diff
@@ -195,15 +195,23 @@ def install_apks(device_serial, app_apk_path, tests_apk_path)
     end
 
     def uninstall_apks(device_serial, app_package_name, tests_package_name)
-      UI.message('Uninstalling app APK')
-      run_adb_command("adb -s #{device_serial} uninstall #{app_package_name}",
-                      print_all: true,
-                      print_command: true)
+      packages = run_adb_command("adb -s #{device_serial} shell pm list packages #{app_package_name}",
+                                 print_all: true,
+                                 print_command: true)
 
-      UI.message('Uninstalling tests APK')
-      run_adb_command("adb -s #{device_serial} uninstall #{tests_package_name}",
-                      print_all: true,
-                      print_command: true)
+      if packages =~ /#{app_package_name}/
```

Comment:
Hm, couldn't this false positive if the `app_package_name` is part of `tests_package_name` (e.g. suffix `.test`)?

Reply:
We have an `AdbHelper` class that could be used to house a new `installed_packages` that returns a list of package names that are installed instead of 1 big string. Would that make sense?

Reply:
@joeboyscout04?

Reply:
It's a great suggestion, sorry I've been away on vacation and holidays until this week. Hopefully will get some time to look at it soon.

Reply:
@janpio I finally got a few free minutes to take another look at this.  I didn't utilize the AdbHelper since it isn't present in the `runner.rb` yet and it didn't feel right introducing it just for this change.  It probably could be part of a future refactoring.

Reply:
Ok, although it would have been fine to introduce the connection to the Helper for that 👍